formatters/simplecov/simplecov.go

```diff
@@ -4,18 +4,39 @@ import (
 	"encoding/json"
 	"os"
 	"sort"
+	"strings"
 	"time"
 
 	"github.com/codeclimate/test-reporter/formatters"
 	"github.com/pkg/errors"
 )
 
+var searchPaths = []string{"coverage/.resultset.json"}
+
 type Formatter struct {
 	Path  string
 	Tests []Test
 }
 
+func (f *Formatter) Search(paths ...string) (string, error) {
+	paths = append(paths, searchPaths...)
+	for _, p := range paths {
+		if _, err := os.Stat(p); err == nil {
+			f.Path = p
+			return p, nil
+		}
+	}
+
+	return "", errors.WithStack(errors.Errorf("could not find a valid file for the simplecov formatter. search paths: %s", strings.Join(paths, ", ")))
```

Comment:
Perhaps:

"Could not find any files in search paths for simplecov. Search paths were:"